eve/main.yml

```diff
@@ -55,3 +68,43 @@ stages:
         command: make lint
         flunkOnFailure: false
         haltOnFailure: false
+
+  single-node:
+    worker:
+      type: openstack
+      image: CentOS 7 (PVHVM)
+      flavor: io1-30
+      path: eve/workers/openstack-single-node
+    steps:
+    - ShellCommand:
+        name: Retrieve ISO image
+        command: >
+          curl -XGET -o metalk8s.iso
+          "%(prop:artifacts_private_url)s/metalk8s.iso"
+        haltOnFailure: true
+    - ShellCommand:
+        name: Retrieve ISO image checksum
+        command: >
+          curl -XGET -o SHA256SUM
+          "%(prop:artifacts_private_url)s/SHA256SUM"
+        haltOnFailure: true
+    - ShellCommand:
+        name: Check image with checksum
+        command: sha256sum -c SHA256SUM
+        haltOnFailure: true
+    - ShellCommand:
+        name: Create mountpoint
+        command: >
+          sudo mkdir -p /srv/scality/metalk8s-%(prop:metalk8s_short_version)s
+    - ShellCommand:
+        name: Mount ISO image
+        command: >
+          sudo mount -o loop metalk8s.iso
+          /srv/scality/metalk8s-%(prop:metalk8s_short_version)s
+        haltOnFailure: true
+    - ShellCommand:
+        name: Start the bootstrap process
+        command: >
+          sudo bash
+          /srv/scality/metalk8s-%(prop:metalk8s_short_version)s/bootstrap.sh
+        haltOnFailure: true
```

Comment:
if you want colors and all on the build output you can set this option `usePTY: true` to your `ShellCommand`

Reply:
Done (c7347ac5b822a1f5b4117c2602f5d342f293cdcd)